middlewares/validators/discordactions.js

```diff
@@ -40,8 +40,24 @@ const validateUpdateUsersNicknameStatusBody = async (req, res, next) => {
   }
 };
 
+export const validateGenerateInviteForUserBody = async (req, res, next) => {
+  const schema = Joi.object().strict().keys({
+    purpose: Joi.string().trim(),
+  });
+
+  try {
+    await schema.validateAsync(req.body);
+    next();
+  } catch (error) {
+    const errorMessages = error.details.map((detail) => detail.message);
+    logger.error(`Error while validating invite creation payload: ${errorMessages}`);
+    res.boom.badRequest(errorMessages);
```

Comment:
use return key word

Reply:
No one has used return in this file for returning error, so I do it ?

Reply:
I have seen return in the same file. Please check it.

Reply:
No, there is no return keyword in this file.